Neos.Flow/Classes/ObjectManagement/DependencyInjection/PropertyInjectionTrait.php

```diff
@@ -27,14 +27,18 @@ trait PropertyInjectionTrait
      * @param callable $lazyInjectionResolver
      * @return void
      */
-    private function Flow_Proxy_LazyPropertyInjection($propertyObjectName, $propertyClassName, $propertyName, $setterArgumentHash, callable $lazyInjectionResolver)
+    private function Flow_Proxy_LazyPropertyInjection(string $propertyObjectName, string $propertyClassName, string $propertyName, string $setterArgumentHash, callable $lazyInjectionResolver)
     {
         $injection_reference = &$this->$propertyName;
         $this->$propertyName = \Neos\Flow\Core\Bootstrap::$staticObjectManager->getInstance($propertyObjectName);
         if ($this->$propertyName === null) {
             $this->$propertyName = \Neos\Flow\Core\Bootstrap::$staticObjectManager->getLazyDependencyByHash($setterArgumentHash, $injection_reference);
             if ($this->$propertyName === null) {
                 $this->$propertyName = \Neos\Flow\Core\Bootstrap::$staticObjectManager->createLazyDependency($setterArgumentHash, $injection_reference, $propertyClassName, $lazyInjectionResolver);
+                if ($this->$propertyName === null) {
```

Comment:
do we allow with this change less lazy dependency injection that before?
related: interfaces fx. cant be injected lazily anymore? https://github.com/neos/flow-development-collection/pull/2701#discussion_r830984712